aea/cli/utils/constants.py

```diff
@@ -66,3 +66,5 @@
     "contracts": DEFAULT_CONTRACT_CONFIG_FILE,
 }  # type: Dict[str, str]
 FALSE_EQUIVALENTS = ["f", "false", "False"]
+
+DEFAULT_AUTHOR = "default_author"
```

Comment:
please remove this constant; it is not needed and should not exist in aea distribution